partition/bootloader_grub.go

```diff
@@ -29,18 +31,14 @@ import (
 
 const (
 	bootloaderGrubDirReal        = "/boot/grub"
```

Comment:
You are removing the other two but not this one. Just sanity question before I understand everything.